### PR TITLE
created FilteringAndSearchingTest class

### PR DESCRIPTION
**Pull request by czajkaaga**
I encourage you to review my code :)

### CODE REVIEW

**Comment by dominikbodzioch** [comment]
In xpath locators we can often use single quotes instead of double quotes (which in java code force to use "escape character"). e.g. 
// a [@ aria-label = 'Toys & Games']
// a [@ aria-label = \\"Toys & Games\\"]
Of course it is not a bug. Just a suggestion :)

As with Marat's code, a nice use of test annotations :) I have to refactor my code in this way myself.
You have already found where the tests do not pass, so I'm waiting for the fixed version :)

**Comment by dominikbodzioch** [comment]
All tests pass!
_SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder"._
I am not sure if it is important. But if you want to avoid this, you should add some dependencies in pom (e.g. available in my pull request).

**Comment by czajkaaga** [comment]
Yes sure, all tests passed so I ignored this message ;)

**Review by dominikbodzioch — APPROVED** [review]
OK

**Review by gercen8905 — COMMENTED** [review]
Great job, Agnieszka! Really nice code!
Just some comments:
1. In your pom you have not necessary imports from 32 to 43. Your code doesn't use them.
2. Call to close() redundant, your tests handle only one window.
3. Naming of test methods could be with expectations like SuccessfulFilteringByPrice() and it will indicate that this test is positive one.
4. 'this' is redundant.
5. chooseCategory() can be private.
6. line 34 or 35 can be deleted.

**Comment by czajkaaga** [comment]
I applied your hints, now you can review my code again :)

**Review by gercen8905 — COMMENTED** [review]
all is fine

File: src/test/java/com/epam/qalab/FilteringAndSearchingTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FilteringAndSearchingTest {
+
+    WebDriver driver;
```
Comment (juliaHrabovska) [inline src/test/java/com/epam/qalab/FilteringAndSearchingTest.java:15]: Can be private

**Pull request merged**